tests/integration-tests/flake8_test.py

```diff
@@ -7,38 +7,11 @@
 from service_messages import ServiceMessage, assert_service_messages, parse_service_messages
 
 
-@pytest.fixture(scope='module', params=["flake8==2.0.0", "flake8==2.4.0", "flake8==2.6.0"])
-def venv_flake8_v2(request):
-    return virtual_environments.prepare_virtualenv([request.param])
-
-
-@pytest.fixture(scope='module', params=["flake8==3.0.3"])
+@pytest.fixture(scope='module', params=["flake8==3.7.9"])
```

Comment:
I think we need to check latest flake8 version anyway. It wan't there before but still

Reply:
done